migration-from-4x.md

```diff
@@ -0,0 +1,328 @@
+
+### Toolbar Menus
+
+* New buttons are added to the global `editor.settings.menus` which is an enhancement since, it now shows the toggled state.
+* Improved mouse and keyboard navigation.
+
+## Plugins
+
+Each release of TinyMCE adds new features and functionality. We also occasionally remove features and functionality, usually because we've added a better option.
+Here are the details about the features and functionalities that we removed in TinyMCE 5.0.
+
+### Removed Features
+
+Plugins with dialogs no longer have `height` or `width` settings for their dialogs. The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions.
+
+The following plugins from TinyMCE 4.x do not require height or width options to be specified in TinyMCE 5.0:
+
+* [Code]({{site.baseurl}}/plugins/code/)
+* [Codesample]({{site.baseurl}}plugins/codesample/)
+* [Preview]({{site.baseurl}}plugins/preview/)
+* [Template]({{site.baseurl}}plugins/template/)
+
+Please read the [docs]({{site.baseurl}}/plugins/) if you need more information on the Tiny 4.x configuration options.
+
+### Changed Features
+
+These features have either changed or have been deleted in TinyMCE 5.0.
+Each release of TinyMCE adds new features and functionality. We also occasionally remove features and functionality, usually because we’ve added a better option. Here are the details about the features and functionalities that we removed in TinyMCE 5.0.
+
+#### Moved Plugins
+
+| **Plugin Name** | **Description** |
+| --------------- |  -------------- |
+| [ContextMenu](https://www.tiny.cloud/docs/plugins/contextmenu/) | New API. See [docs]({{site.baseurl}}/components/contextmenu/). |
+
+### Table
+
+Changes between TinyMCE 4.x and TinyMCE 5.0:
+
+* Styles text field has been removed from the advanced table of the dialogs. This simplifies the dialogs for users and gives the editor stricter control over the table styles which means we are better able to ensure the styles are correct.
+* Improved how styles are set and retrieved from tables, rows, and cells, so this should be more reliable now.
+* Shifted to using CSS more for styling, and therefore removed a few legacy data attributes that were set on tables/rows/cells which are no longer the good practice to use. This makes the output HTML cleaner and more modern.
+* When opening a properties dialog with a single table/row/cell selected, the dialog will autofill with the relevant existing values. If you select multiple rows or cells and open the relevant properties dialog, TinyMCE 4.x will leave all the dialog fields blank. In TinyMCE 5.0, any fields which have the same values for all the selected rows or cells will autofill, and the fields which have no existing value or have differing values will be empty.
+* `Border` input field in the `tableprops` dialog is now called `Border width`, for better clarity.
+
+
+## Mobile Support
```

Comment:
@EkimChau Please confirm this section is still correct.

Reply:
Confirmed with @blackcathikari. It's correct